deployer/src/lib.rs

```diff
@@ -54,12 +55,15 @@ pub async fn start(
     );
     let make_service = router.into_make_service();
 
-    info!("Binding to and listening at address: {}", args.api_address);
-
-    axum::Server::bind(&args.api_address)
-        .serve(make_service)
-        .await
-        .unwrap_or_else(|_| panic!("Failed to bind to address: {}", args.api_address));
+    select! {
+        _ = tokio::spawn(shuttle_runtime::start_legacy()) => {
+            info!("Legacy runtime stopped.")
```

Comment:
So this would be some [`Command::new()`](https://doc.rust-lang.org/std/process/struct.Command.html) to start the binary in a new process.